CHANGELOG.md

```diff
@@ -76,6 +76,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Bug Fixes
 
+* Fix set cache_root from config file error  
```

Comment:
Please put this in the `Master` section?

Reply:
done